src/test/java/com/wikia/webdriver/testcases/logintests/LoginTests.java

```diff
@@ -49,4 +49,11 @@ public void Login_004_dropDownStaff() {
     dropDown.logIn(credentials.userNameStaff, credentials.passwordStaff);
     base.verifyUserLoggedIn(credentials.userNameStaff);
   }
+
+  @Test(groups = {"Login_005", "Login"})
+  public void Login_005_japaneseUserLogin() {
```

Comment:
you can annotate the test
@Execute(asUser = User.userNameJapanese2, onWikia = jaTestWiki)